themes/theme-b2b/src/tokens.ts

```diff
@@ -27,6 +27,22 @@ const brand = {
     '950': '#030712',
   },
 
+  white: '#ffff',
```

Comment:
Is this different from the tw one? Should we explicitly add these 🤔

Reply:
what do you mean?

Reply:
@sebald

Reply:
this line basically does nothing, because `bg-white` is already there in tailwind :) So my question is if we add this explicitly.